.ci/lib.sh

```diff
@@ -374,9 +373,7 @@ gen_clean_arch() {
 
 		info "Clean up stale network interface"
```

Comment:
disable that part

Reply:
lets see if that line fixes the issue. thanks